linkerd/docs/interpreter.md

```diff
@@ -63,6 +60,42 @@ Key | Default Value | Description
 commonName | _required_ | The common name to use for namerd requests.
 caCert | N/A | The path to the CA cert used for common name validation.
 
+## namerd http
+
+kind: `io.l5d.namerd.http`
+
+The namerd http interpreter offloads the responsibilities of name resolution to
+the namerd service.  Any namers configured in this linkerd are not used.  The
+interpreter uses namerd's HTTP streaming interface.  Note that the protocol that
+the interpreter uses to talk to namerd is unrelated to the protocols of
+linkerd's routers.
+
+Key | Default Value | Description
+--- | ------------- | -----------
+experimental | _required_ | Because the http interpreter is still considered experimental, you must set this to `true` to use it.
+dst | _required_ | A Finagle path locating the namerd service.
+namespace | `default` | The name of the namerd dtab to use.
+retry | see [namerd retry](#namerd-retry) | An object configuring retry backoffs for requests to namerd.
+tls | no tls | Requests to namerd will be made using TLS if this parameter is provided.  It must be a [namerd client TLS](#namerd-client-tls) object.
+
+## namerd mesh
+
+kind: `io.l5d.mesh`
+
+The namerd mesh interpreter offloads the responsibilities of name resolution to
+the namerd service.  Any namers configured in this linkerd are not used.  The
+interpreter uses namerd's mesh interface via gRPC.  Note that the protocol that
+the interpreter uses to talk to namerd is unrelated to the protocols of
+linkerd's routers.
+
+Key | Default Value | Description
+--- | ------------- | -----------
+experimental | _required_ | Because the mesh interpreter is still considered experimental, you must set this to `true` to use it.
+dst | _required_ | A Finagle path locating the namerd service.
+root | `/default` | A single-element Finagle path representing the namerd namespace.
+retry | see [namerd retry](#namerd-retry) | An object configuring retry backoffs for requests to namerd.
+tls | no tls | The namerd mesh interface does not support server TLS configuration at this time.
```

Comment:
Yep, TLS is configurable on the client, but the server doesn't support it yet, so there's no point in setting it here. I wanted to document that so that folks didn't see the option and try to use it. Tracking TLS server support in #1191.